appOPHD/States/MapViewStateTurn.cpp

```diff
@@ -359,9 +359,9 @@ void MapViewState::findMineRoutes()
 void MapViewState::transportOreFromMines()
 {
 	auto& routeTable = NAS2D::Utility<std::map<class MineFacility*, Route>>::get();
-	for (auto* mine : NAS2D::Utility<StructureManager>::get().getStructures<MineFacility>())
+	for (auto* mineFacilityPtr : NAS2D::Utility<StructureManager>::get().getStructures<MineFacility>())
 	{
-		auto routeIt = routeTable.find(mine);
+		auto routeIt = routeTable.find(mineFacilityPtr);
```

Comment:
This section is a little inconsistent with the `mineFacilityPtr` name compared to the last section which used `mineFacility`. I think I might prefer leaving "Ptr" out of the name.